es-core/src/components/ImageGridComponent.h

```diff
@@ -96,7 +101,21 @@ void ImageGridComponent<T>::add(const std::string& name, const std::string& imag
 	typename IList<ImageGridData, T>::Entry entry;
 	entry.name = name;
 	entry.object = obj;
-	entry.data.texture = ResourceManager::getInstance()->fileExists(imagePath) ? TextureResource::get(imagePath) : TextureResource::get(":/button.png");
+
+	// FileType::FOLDER = 2, but FileData is our template parameter T,
+	// so we don't want to bring that dependence to FileData here
+	if (obj->getType() == 2)
+	{
+		entry.data.texture = mFolderTexture;
```

Comment:
Folders can have images specified in gamelist.xml  We should probably use that if present vs. defaulting to folder image.